Privacy policy (#138)
mn-pollinators/buzz-about: docs/privacy-policy.md

# Privacy Policy

*Effective 2020-10-05*

## Collection of Information

### Information you give
Buzz About receives personal information when you sign in. This may include your email address and your phone number.

### Information we collect automatically
We collect information about your interactions in Buzz About. While Buzz About itself does not collect any information automatically about your machine, third party services that we use may collect that information. Check the privacy policies of our third party services below to read about the information they collect.

## Use of Information

> The data collected by Buzz About is used by a research team at the University of Minnesota Morris. The purpose of this research is to learn more about how taking part in activities that show what bees do and what they need might help people understand more about how to support bees and how they help people. We want to know more about how we can use technology to help people learn more from these activities. The results of this research might help people design technology in a way that helps people learn more effectively. Along with the current Buzz About research, the data collected as part of Buzz About may be used in later studies.

Reviewer: ```suggestion
The data collected by Buzz About is used by a research team at the University of Minnesota Morris. The purpose of this research is to learn more about how taking part in activities that show what bees do and what they need might help people understand more about how to support bees and how they help people. We want to know more about how we can use technology to help people learn more from these activities. The results of this research might help people design technology in a way that helps people learn more effectively. Along with current Buzz About research, the data collected as part of Buzz About may be used in later studies.
```

Also, how do you feel about "the data we collect" instead of "the data collected as part of Buzz About"?

Author: Sure

> <p> In line with ethical guidelines for research, efforts will be made to limit the use of any data collected by Buzz About, including personal information, to the people who need to view the information. This may include the [University of Minnesota Institutional Review Board](https://research.umn.edu/units/irb) and other representatives of the institution.

Reviewer: I'm not sure if you meant to put this extra `<p>` tag here but it's breaking the markdown since it isn't closed.
```suggestion
In line with ethical guidelines for research, efforts will be made to limit the use of any data collected by Buzz About, including personal information, to the people who need to view the information. This may include the [University of Minnesota Institutional Review Board](https://research.umn.edu/units/irb) and other representatives of the institution.
```

Also if you want to break this into two paragraphs you can add an empty line between them:
```suggestion

In line with ethical guidelines for research, efforts will be made to limit the use of any data collected by Buzz About, including personal information, to the people who need to view the information. This may include the [University of Minnesota Institutional Review Board](https://research.umn.edu/units/irb) and other representatives of the institution.
```

# Privacy Policy

> *Effective 2020-10-05*

Reviewer: Should we change this date to be more recent since we have been changing the policy?

## Collection of Information

### Information you give
Buzz About receives personal information when you sign in. This may include your email address and your phone number.

### Information we collect automatically

> We collect information about your interactions in Buzz About. While Buzz About itself does not collect any information automatically about your machine, third party services that we use may collect that information. Check the privacy policies of our third party services below to read about the information they collect.

Reviewer 2: ```suggestion
We collect information about your interactions in Buzz About. While Buzz About itself does not collect any information automatically about your computer or device, third party services that we use may collect that information. Check the privacy policies of our third party services below to read about the information they collect.
```

Reviewer: ```suggestion
We collect information about your interactions in Buzz About. While Buzz About itself does not collect any information automatically about your device, third party services that we use may collect that information. Check the privacy policies of our third party services below to read about the information they collect.
```

I would prefer just saying "device". Good catch KK, something like "device" is far more common for this type of writing than "machine"

# Privacy Policy

*Effective 2020-10-05*

## Collection of Information

### Information you give
Buzz About receives personal information when you sign in. This may include your email address and your phone number.

### Information we collect automatically
We collect information about your interactions in Buzz About. While Buzz About itself does not collect any information automatically about your machine, third party services that we use may collect that information. Check the privacy policies of our third party services below to read about the information they collect.

## Use of Information
The data collected by Buzz About is used by a research team at the University of Minnesota Morris. The purpose of this research is to learn more about how taking part in activities that show what bees do and what they need might help people understand more about how to support bees and how they help people. We want to know more about how we can use technology to help people learn more from these activities. The results of this research might help people design technology in a way that helps people learn more effectively.

> In line with ethical guidelines for research, efforts will be made to limit the use of any data collected by Buzz About, including personal information, to the people who need to view the information. This may include the [University of Minnesota Institutional Review Board](https://research.umn.edu/units/irb) and other members of the institution.</p>

Reviewer: ```suggestion
In line with ethical guidelines for research, efforts will be made to limit the use of any data collected by Buzz About, including personal information, to the people who need to view the information. This may include the [University of Minnesota Institutional Review Board](https://research.umn.edu/units/irb) and other members of the institution.
```

## Children’s Privacy
In compliance with the Children’s Online Privacy Protection Act (COPPA), Buzz About does not knowingly collect, use, or distribute personal information from children under the age of 18 without prior verifiable consent from a parent or guardian. Additionally, Buzz About does not ask children under the age of 18 to divulge more information than is needed to participate in the activity.

## International Users
Buzz About was developed and is maintained in the United States, which may not provide the same level of protection as the laws in your jurisdiction. By using Buzz About, you understand and agree that your information may be transferred to and stored on servers located outside of your resident jurisdiction. Furthermore, Buzz About is not responsible for the data’s storage once it has been transferred to a third party service.

## Changes to this Policy
We may modify this privacy policy at any time by posting revisions to this document. When we update the privacy policy, we will revise the "Effective Date" above. In all cases, your continued use of Buzz About indicates your assent to the amended privacy policy.

## Contact
  > Kristin Lamberty
  >

>   > University of Minnesota - Morris

Reviewer: ```suggestion
  > University of Minnesota Morris
```

>   > Kristin Lamberty
>   >
>   > University of Minnesota - Morris
>   >
>   > Email Address: <mn-polinators@morris.umn.edu>

Reviewer: I don't think this makes sense as a quote. 
```suggestion

Kristin Lamberty

University of Minnesota - Morris

Email Address: <mn-polinators@morris.umn.edu>
```
